Dockerfile

```diff
@@ -200,6 +200,8 @@ CMD ["run"]
 # HTTP is 8080, JMX is 9001, prometheus is 9090, Hazelcast is 5701-5710, Ignite is 47100, REST for Kafka is 7003
 EXPOSE 8080 9001 9090 5701-5710 47100 7003
 
+HEALTHCHECK --interval=5m --timeout=3s CMD curl -f http://localhost/ || exit 1
```

Comment:
Should we add a bit of commentary to this check, since it's odd for us to include a docker healthcheck? Something like:

```
# Necessary for CIS Level 2 compliance if this image is used outside of a Kubernetes context.
# Kubernetes ignores this check and uses the liveness/readiness probes instead.
```

Reply:
added